docs/userguide.md

```diff
@@ -321,8 +321,8 @@ successfully verified a working federation-v2 deployment.
 To cleanup the example simply delete the namespace and its placement:
 
 ```bash
+kubectl -n test-namespace delete federatednamespaceplacement test-namespace
```

Comment:
do we still need this step? I think `kubectl delete ns test-namespace` will delete all resources under this namespace including the namespace placement.

Reply:
This is intended to work with v0.0.1 as well as the latest master. In v0.0.1, `federatednamespaceplacement` is cluster-scoped so this deletion still needs to happen and the `test-namespace` option will be ignored.

Reply:
> This is intended to work with v0.0.1 as well as the latest master. 
Yes, but I think the document here is intend for 0.0.2 and future versions, why still need to consider 0.0.1?

Reply:
The document is still targeting 0.0.1 because the `install-latest.yaml` is still referencing the latest tag. So until we update the config to use the latest master, this step should remain.